lxd/instance/drivers/driver_lxc.go

```diff
@@ -21,7 +21,7 @@ import (
 	"github.com/flosch/pongo2"
 	"github.com/pkg/errors"
 	"golang.org/x/sys/unix"
-	lxc "gopkg.in/lxc/go-lxc.v2"
+	liblxc "gopkg.in/lxc/go-lxc.v2"
```

Comment:
Are you doing this in the rest of the codebase (if there's any other import sites)?

Reply:
@stgraber I'll check and update where needed.